docs/.vuepress/theme/components/Footer.vue

```diff
@@ -96,65 +58,232 @@ export default {
+              text: 'Ethereum Wallets',
+              useRouter: true
+            },
+            {
+              to: '/learn/',
+              text: 'Guides & Resources',
+              useRouter: true
+            },
+            {
+              to: '/community/',
+              text: 'Ethereum Community',
+              useRouter: true
+            }
+          ]
+        },
+        {
+          title: 'Developers',
+          items: [
+            {
+              to: '/build/',
+              text: 'Get Started',
+              useRouter: true
+            },
+            {
+              to: 'https://studio.ethereum.org/',
+              text: 'Ethereum Studio',
+              newTab: true
+            },
+            {
+              to: '/developers/',
+              text: 'Developer Resources',
+              useRouter: true
+            }
+          ]
+        },
+        {
+          title: 'Ecosystem',
+          items: [
+            {
+              to: 'https://blog.ethereum.org/',
+              text: 'Blog',
+              newTab: true
+            },
+            {
+              to: 'https://ecosystem.support',
+              text: 'Ecosystem Support Program',
+              newTab: true
+            },
+            {
+              to: 'https://eips.ethereum.org/',
+              text: 'EIPs',
+              newTab: true
+            },
+            {
+              to: '/assets/',
+              text: 'Ethereum Brand Assets',
+              useRouter: true
+            }
+          ]
+        },
+        {
+          title: 'About ethereum.org',
+          items: [
+            {
+              to: 'https://blog.ethereum.org/',
+              text: 'Blog',
+              newTab: true
+            },
+            {
+              to: 'https://ecosystem.support',
+              text: 'Ecosystem Support Program',
+              newTab: true
+            },
+            {
+              to: 'https://eips.ethereum.org/',
+              text: 'EIPss',
+              newTab: true
+            },
+            {
+              to: '/assets/',
+              text: 'Ethereum Brand Assets',
+              useRouter: true
+            }
+          ]
+        }
+      ]
+    }
   }
 }
 </script>
 
-<style lang="stylus" scoped>
-@require '../styles/config'
+<style lang="stylus">
+@require '../styles/config';
+
+// move forwards with json objects or CSS vars
+json('../styles/media-queries.json');
 
 footer
-  font-size $fsSmall
-  padding 6em 2em 2em 10em
+  // use p.small font-size
+  font-size .875rem
+  line-height 1.4em
+  box-sizing border-box
+  width 85vw
+  max-width 1440px
+  margin 0 auto
+  padding-top 48px
+  padding-bottom 64px
+  display flex
+  flex-wrap wrap
+  justify-content space-between
 
-  div.credits
-    display none
-    margin-bottom 0.5em
-    color $subduedColor
+  // Set padding at breakpoints
+  @media L
+    padding-bottom 128px
+    padding-top 64px
+  @media M
+    padding 80px inherit 160px
```

Comment:
Is this legit? My browser shows this as invalid:
<img width="307" alt="Image 2020-02-28 at 12 12 26 PM" src="https://user-images.githubusercontent.com/8097623/75547792-a497fa00-5a23-11ea-8a4c-8c133abd3542.png">

Reply:
oh weird! this was working. let me double check

Reply:
removed this